### DISCUSSION TITLE
How to make swagger ui show authentication token field like how postman does for fastapi

### DISCUSSION THREAD

**Discussion by rohit-ascendo** [fastapi/fastapi#4471] [Questions] [answered]
Hi, i want to be able to send authentication tokens in headers for fastapi swagger apis, right now i have to use postman.
I was looking at fastapi docs for extending swagger ui but i have no idea how to do this . please help

**Comment by YuriiMotov** [comment, marked as answer]
https://fastapi.tiangolo.com/tutorial/security/first-steps/
